polus-basic-flatfield-correction-plugin/src/main.py

```diff
@@ -56,50 +56,39 @@ def main():
     logger.info('get_photobleach = {}'.format(get_photobleach))
     logger.info('inp_regex = {}'.format(inp_regex))
     logger.info('output_dir = {}'.format(output_dir))
```

Comment:
The `input_dir` needs to have some variable checking performed on it. If there is a subdirectory named `images`, then you need to replace `input_dir` with `{input_dir}/images`.